src/Contrib/Jaeger/CodecUtility.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\Jaeger;
+
+use Throwable;
+
+class CodecUtility
```

Comment:
A lot of this code exists already (or is very similar) in `SDK\Trace\RandomIdGenerator`, so I think we should use that, and add in any extra functionality from this class.

Reply:
Yeah I couldn't actually find any usages of this class. I think I sorta see what the first method was doing but yeah not so sure about this trace Id generation part